specification/nodes/info/nodesInfoResponseExample1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "summary": "A successful response when requesting cluster nodes information.",
+  "description": "",
+  "type": "response",
+  "response_code": 200,
+  "value": "{\n  \"_nodes\": ...\n  \"cluster_name\": \"elasticsearch\",\n  \"nodes\": {\n    \"USpTGYaBSIKbgSUJR2Z9lg\": {\n      \"name\": \"node-0\",\n      \"transport_address\": \"192.168.17:9300\",\n      \"host\": \"node-0.elastic.co\",\n      \"ip\": \"192.168.17\",\n      \"version\": \"{version}\",\n      \"transport_version\": 100000298,\n      \"index_version\": 100000074,\n      \"component_versions\": {\n        \"ml_config_version\": 100000162,\n        \"transform_config_version\": 100000096\n      },\n      \"build_flavor\": \"default\",\n      \"build_type\": \"{build_type}\",\n      \"build_hash\": \"587409e\",\n      \"roles\": [\n        \"master\",\n        \"data\",\n        \"ingest\"\n      ],\n      \"attributes\": {},\n      \"plugins\": [\n        {\n          \"name\": \"analysis-icu\",\n          \"version\": \"{version}\",\n          \"description\": \"The ICU Analysis plugin integrates Lucene ICU module into elasticsearch, adding ICU relates analysis components.\",\n          \"classname\": \"org.elasticsearch.plugin.analysis.icu.AnalysisICUPlugin\",\n          \"has_native_controller\": false\n        }\n      ],\n      \"modules\": [\n        {\n          \"name\": \"lang-painless\",\n          \"version\": \"{version}\",\n          \"description\": \"An easy, safe and fast scripting language for Elasticsearch\",\n          \"classname\": \"org.elasticsearch.painless.PainlessPlugin\",\n          \"has_native_controller\": false\n        }\n      ]\n    }\n  }\n}"
```

Comment:
>  "value": "{\n  \"_nodes\": ...\n  \"cluster_name\": \"elasticsearch\",\n  \"nodes\": {\n    

I think this one will fail because it's not a complete example ("...").

Reply:
I modified the key:value pair to form a valid JSON. Is it good enough do you think for this particular example?

Reply:
That seems to work, thanks (though I needed to add two more escape characters around the quotes for it to work).